### PR TITLE
(phi0690)-phi0690.phi001-phi0690.phi003.perseus-eng2/lat2_cts.xml

### LINKED ISSUE #430: Misattribution of translation - Vergil's Aeneid
Hi!

I'm new to GitHub so I'm not sure how to log this but I was looking for a translation of Vergil's Aeneid and it seems to have been misattributed. The Scaife viewer version seems to be the Theodore C Williams version from 1910, but it's been attributed to John Dryden (1697). Both versions exist in the Perseus Tufts database. I've attached screenshots of the first few lines from both for comparison. Thought I'd let you know!

Thanks. (:

Screenshot 2021-06-03 at 11 45 08
Screenshot 2021-06-03 at 11 46 16
Screenshot 2021-06-03 at 11 46 30
Screenshot 2021-06-03 at 11 46 43

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Corrected _cts.xml for phi0690.phi003.perseus-eng2 also updated the file header to current standards.
Updated headers for both phi0690.phi003.perseus-lat2 and perseus-eng2

Fix #430 

Per issue #431.

### CODE REVIEW

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5472)*

File: data/phi0690/phi003/phi0690.phi003.perseus-lat2.xml [hunk 1]
```diff
@@ -36,18 +35,15 @@
                 <biblStruct>
                     <monogr>
                         <author>Vergil</author>
-                        <author>Virgil</author>
                         <title>The Bucolics, Aeneid, and Georgics Of Virgil</title>
                         <editor role="editor">J. B. Greenough</editor>
```
Comment (lcerrato) [inline data/phi0690/phi003/phi0690.phi003.perseus-lat2.xml:39]: `<editor role="editor">` s/b `<editor>`

File: data/phi0690/phi003/phi0690.phi003.perseus-eng2.xml [hunk 2]
```diff
@@ -38,17 +37,18 @@
 	           <biblStruct>
 	              <monogr>
 	                 <author>Vergil</author>
-	                 <title>Aeneid</title>
+	                 <title>The Aeneid of Virgil</title>
 	                 <respStmt>
 	                    <name>Theodore C. Williams</name>
-	                    <resp>trans.</resp>
+	                    <resp>translator</resp>
```
Comment (lcerrato) [inline data/phi0690/phi003/phi0690.phi003.perseus-eng2.xml:43]: Probably preferable practice to use `<editor role="translator">` given what Eldarion is doing with `<respStmt>` tags

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/phi0690/phi003/phi0690.phi003.perseus-lat2.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/phi0690/phi003/phi0690.phi003.perseus-lat2.xml:39]: I'm not quite sure what you mean by s/b @lcerrato sorry if I am missing the obvious.

Comment (lcerrato) [reply data/phi0690/phi003/phi0690.phi003.perseus-lat2.xml:39]: Sorry, should be

Comment (AlisonBabeu) [reply data/phi0690/phi003/phi0690.phi003.perseus-lat2.xml:39]: So its ok that I took out the role="editor" right that's what I have been doing.

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5474)*

**Review by lcerrato — APPROVED** [review]

**Pull request merged**